### PR TITLE
[#164119920, #164024212] audio integration updates for iOS

### PR DESCRIPTION
**Pull request by leoleblanc**
Audio Integration changes for iOS CaveRunner.

Feature is fully functional and changes in volume persist through subsequent games
image from ios

### CODE REVIEW

File: iOS/3D Cave Runner Xcode/Classes/Unity/WWWConnection.mm [hunk 1]
```diff
@@ -40,11 +39,14 @@ @implementation UnityWWWConnectionDelegate
     NSString*           _password;
 
     // response
-    NSString*           _responseHeader;
     NSInteger           _status;
     size_t              _estimatedLength;
     size_t              _dataRecievd;
     int                 _retryCount;
+    NSOutputStream*     _outputStream;
+
```
Comment (niemp296) [inline iOS/3D Cave Runner Xcode/Classes/Unity/WWWConnection.mm]: should you remove empty line here?

File: iOS/3D Cave Runner Xcode/Classes/Unity/WWWConnection.mm [hunk 2]
```diff
@@ -54,6 +56,7 @@ @implementation UnityWWWConnectionDelegate
 @synthesize connection  = _connection;
```
Comment (niemp296) [inline iOS/3D Cave Runner Xcode/Classes/Unity/WWWConnection.mm]: empty line here as well

File: iOS/3D Cave Runner Xcode/Classes/UI/SplashScreen.mm [hunk 3]
```diff
@@ -45,84 +46,76 @@
...
-        if (orientPortrait && rotateToPortrait)
-            return "-Portrait";
-        else if (orientLandscape && rotateToLandscape)
-            return "-Landscape";
-        else if (rotateToPortrait)
-            return "-Portrait";
-        else
-            return "-Landscape";
-    }
-    return "";
```
Comment (niemp296) [inline iOS/3D Cave Runner Xcode/Classes/UI/SplashScreen.mm]: why don't you need to check if it is orientable anymore? what happen if the game is force to always be a specific orientation?

**Comment by leoleblanc** [comment]
All of the changes you're suggesting are inside of Unity-specific library files.  They appear to be following their own conventions, but I can provide reasons for each question you have

**Comment by niemp296** [comment]
👍

**Comment by niemp296** [comment]
👍

**Comment by gabrielfirmacion** [comment]
👍

**Pull request merged**